nixos/modules/services/audio/spotifyd.nix

```diff
@@ -27,6 +27,7 @@ in
       wantedBy = [ "multi-user.target" ];
       after = [ "network-online.target" "sound.target" ];
       description = "spotifyd, a Spotify playing daemon";
+      environment.SHELL = "/bin/sh";
```

Comment:
I think `"${pkgs.runtimeShell}/bin/sh"` would be more appropriate. But I'm not sure.